scripts/build_native_deps.sh

```diff
@@ -378,6 +397,13 @@ build_harfbuzz()
     # Don't build test and docs
     sed_i "s|SUBDIRS = src util test docs|SUBDIRS = src util|" "$src/Makefile.in"
 
+    if [ "$uname" = "Darwin" ]; then
+        export LDFLAGS="-framework Foundation"
+
+        # TODO: This feels fishy. Investigate further.
+        sed_i "s/@HAVE_CORETEXT_TRUE@//" "$src/src/Makefile.in"
+    fi
```

Comment:
Yes, this part looks indeed fishy. Which errors do you get without `LDFLAGS` and `sed_i`?

Reply:
No build errors, but one of the Pango unit tests failed, because it could not find `hb_coretext.h`. I'll remove this and upload the error log later.

Reply:
… or if you prefer to not build tests, we can just do that for Pango as well and get rid of this fishy line.

Reply:
Hmm, I don't see that the `tests` subdirectory is included anywhere, and I don't have it in my `build` directory. So I'm currently a bit confused that you have problems with unit tests